plugins/arDominionPlugin/css/less/jobs.less

```diff
@@ -48,7 +48,7 @@
   margin: 5px 5px 0px 5px;
   padding: 5px;
   background-color: #f2f2f2;
-  border-radius: 10px;
+  border-radius: 4px;
   color: @black;
   font-family: Courier New;
```

Comment:
Don't do that, unless it's really necessary. Our theme sets <pre> to a lesscss variable called `@monoFontFamily ` (see https://github.com/artefactual/atom/blob/qa/2.4.x/plugins/arDominionPlugin/css/less/variables.less#L71), which equals to `Monaco, Menlo, Consolas, "Courier New", monospace`, which provides better consistency across multiple browsers and platforms.